### DISCUSSION TITLE
1M+ rows demo

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#4817] [Ideas]
Just logging this as a random idea but to show the robustness of what we've built, we should add an object in the demo account (not seed) with 1M+ rows. I don't think there should be any performance issue compared to 10k rows (frontend performance while loading many rows isn't great though), and it would be re-assuring for users to see it works well.

Not a priority for now

**Comment by BOHEUS** [comment]
I run some tests and when there are 250k objects, search takes on average ~115ms, with 660k objects it increases up to 280-300ms according to logs, I'm assuming it's logarithmic time and when there are 1M objects, it can take ~500ms on average. I'm attaching the video how it looks with 660k rows for end-user
[Screencast from 2024-08-25 15-15-24.webm](https://github.com/user-attachments/assets/e5f6e5b0-b01a-4667-9b96-f239b729522a)
